src/client/src/views/Scenes.js

```diff
@@ -18,11 +18,9 @@ module.exports = {
           return m('div.scene-list-item', [
             m('a', {
               onclick: function() {
-                m.route.set(`/edit-scene/${scene.id}`, {
-                  captions: scene.captions
-                })
+                m.route.set(`/edit-scene/${scene.id}`)
```

Comment:
Can we maintain the ability to click the entire box for entering a caption list for a selected scene? These new changes force you to click the name

Reply:
Technically that is a change with the addition of the delete button (since the delete button is within the box).

You might be right though that it would be better to allow the user to click anywhere in the box.

The commits that changed that have already been merged in, so that particular concern isn't an issue with this PR as far as I know.

I can create a backlog item for that though.